plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestCachingJdbcClient.java

```diff
@@ -198,19 +196,19 @@ public void testColumnsCachedPerSession()
         JdbcTableHandle table = getAnyTable(schema);
         JdbcColumnHandle phantomColumn = addColumn(table);
 
-        int expectedLoad = 0, expectedHit = 0, expectedMiss = 0;
+        AssertedStats stats = new AssertedStats();
```

Comment:
That should get `cachingJdbcClient.getColumnsCacheStats()` as parameter.

```
assertStats = asserStats(cachingJdbcClient.getColumnsCacheStats());
assertStats.addLoaded(1).addMisses(1);
```

Each `add*` could do already assertion..

Reply:
PTAL now. I've made a fixup that I will squash into final form if we agree on solution.